libs/icu/release-72-1/wl-build.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+logStatus "Building libs 'icu/release-72-1'"
+
+if [[ ! -v WASMLABS_ENV ]]
+then
+    echo "Wasmlabs environment is not set"
+    exit 1
+fi
+
+# export CFLAGS_CONFIG="-O3 -g"
+export CFLAGS_CONFIG="-O0"
+
+export CFLAGS_WASI="--sysroot=${WASI_SYSROOT} -I./wasmlabs-stubs -D_WASI_EMULATED_MMAN -D_WASI_EMULATED_GETPID -D_WASI_EMULATED_SIGNAL -D_WASI_EMULATED_PROCESS_CLOCKS"
+export LDFLAGS_WASI="--sysroot=${WASI_SYSROOT} -lwasi-emulated-mman -lwasi-emulated-getpid -lwasi-emulated-signal -lwasi-emulated-process-clocks"
+
+export CFLAGS_ICU=''
+
+logStatus "Using ICU DEFINES: ${CFLAGS_ICU}"
+
+export CFLAGS_BUILD='-D_POSIX_SOURCE=1 -D_GNU_SOURCE=1 -DHAVE_FORK=0 -DWASM_WASI'
```

Comment:
We can get rid of `WASM_WASI`, right?

Reply:
Yes! `__wasi__` is automatically defined, and we should use it in the patches.

Even, @gzurl if you have the time to experiment I would try removing as many of these flags as possible. Including the WASI ones for emulated stuff.

Reply:
+1, also especially given we are not patching the source.

Reply:
Will do!